monkey/monkey_island/cc/ui/src/components/pages/MapPage.js

```diff
@@ -138,16 +150,37 @@ class MapPageComponent extends AuthComponent {
     );
   }
 
+  handleScroll(e) {
+    let element = e.target;
+
+    let telemetryStyle = window.getComputedStyle(element)
+    let telemetryLineHeight = parseInt((telemetryStyle.lineHeight).replace('px', ''))
+
+    this.setState({
+      isScrolledUp: !!(element.scrollTop < this.scrollTop),
```

Comment:
It is so nice how you solved the setState and redundancy problem with a single statement.  But I think you probably left `!!` by mistake.

Reply:
Agree - can be simplified:

```suggestion
      isScrolledUp: (element.scrollTop < this.scrollTop),
```